clients/fides-js/src/lib/a11y-dialog.tsx

```diff
@@ -6,21 +6,26 @@
 import A11yDialogLib from "a11y-dialog";
 import { useCallback, useEffect, useState } from "preact/hooks";
 
-const useA11yDialogInstance = () => {
+const useA11yDialogInstance = (addOverlowStyling: Boolean) => {
```

Comment:
I'm OK with this as a test fix; but it's really narrow and still leaves behind an `a11y-dialog` instance when we _aren't_ actually showing a dialog.

Can you create a new PROD bug and assign to current sprint like "Fix a11y issues with embedded modal" to follow up further here? Because right now it has some issues:
* appears to automatically grab focus on open, which can force a jarring page scroll
* labeled as `role="alertdialog"` and `aria-modal="true"` which will likely confuse a screen reader / etc.
* basically just feels wrong 👍

Reply:
https://ethyca.atlassian.net/browse/PROD-1277